### PR TITLE
Convert FindRefs command to ROP

### PR DESCRIPTION
**Pull request by mexx**
related to #464

### CODE REVIEW

**Comment by forki** [comment]
LGTM

File: src/Paket.Core/LockFile.fs [hunk 1]
```diff
@@ -284,6 +284,27 @@ module LockFileParser =
 /// Allows to parse and analyze paket.lock files.
 type LockFile(fileName:string,options,resolution:PackageResolution,remoteFiles:ResolvedSourceFile list) =
 
+    let dependenciesByPackageLazy = lazy (
```
Comment (theimowski) [inline src/Paket.Core/LockFile.fs]: Why lazy? Can't be just a ``unit -> map`` function?

**Comment by theimowski** [comment]
nice :+1:

File: src/Paket.Core/LockFile.fs (hunk 1 above)
Comment (mexx) [inline src/Paket.Core/LockFile.fs]: Memorization, it only needs to be calculated once.

**Pull request merged**